README.md

```diff
@@ -1,4 +1,4 @@
-# Test-Driven Development
+nod# Test-Driven Development
 
```

Comment:
Did you include your domain model?

Reply:
As Arisa, suggested it would be good to see your domain model 👍

Reply:
I did a domain model in a spreadsheet.